packages/babel-plugin-transform-es2015-typeof-symbol/src/index.js

```diff
@@ -3,6 +3,14 @@ export default function ({ types: t }) {
 
   return {
     visitor: {
+      Scope({ scope }) {
+        if (!scope.getBinding("Symbol")) {
+          return;
+        }
+
+        scope.rename("Symbol");
```

Comment:
If someone imported their Promise lib as `Promise` it would cause the same error Lodash was seeing with its `Symbol` module.